packer/cache_config_unix.go

```diff
@@ -0,0 +1,21 @@
+// +build darwin freebsd linux netbsd openbsd solaris
+
+package packer
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func getDefaultCacheDir() string {
+	var defaultConfigFileDir string
+
+	if xdgCacheHome := os.Getenv("XDG_CACHE_HOME"); xdgCacheHome != "" {
+		defaultConfigFileDir = filepath.Join(xdgCacheHome, "packer")
+	} else {
```

Comment:
NIT: same here:
```suggestion
	if xdgCacheHome := os.Getenv("XDG_CACHE_HOME"); xdgCacheHome != "" {
		return filepath.Join(xdgCacheHome, "packer")
	}
```
(this won't compile)